Classes/DDLoggerNames.m

```diff
@@ -0,0 +1,21 @@
+// Software License Agreement (BSD License)
+//
+// Copyright (c) 2010-2018, Deusty, LLC
+// All rights reserved.
+//
+// Redistribution and use of this software in source and binary forms,
+// with or without modification, are permitted provided that the following conditions are met:
+//
+// * Redistributions of source code must retain the above copyright notice,
+//   this list of conditions and the following disclaimer.
+//
+// * Neither the name of Deusty nor the names of its contributors may be used
+//   to endorse or promote products derived from this software without specific
+//   prior written permission of Deusty, LLC.
+
+#import "DDLoggerNames.h"
+
+DDLoggerName DDLoggerNameASL    = @"cocoa.lumberjack.aslLogger";    // DDASLLogger
+DDLoggerName DDLoggerNameTTY    = @"cocoa.lumberjack.ttyLogger";    // DDTTYLogger
+DDLoggerName DDLoggerNameOS     = @"cocoa.lumberjack.osLogger";     // DDOSLogger
+DDLoggerName DDLoggerNameFile   = @"cocoa.lumberjack.fileLogger";   // DDOSLogger
```

Comment:
same here

Reply:
Maybe leave only one logger?

Reply:
Yep, I don't think we need the comments in the implementation files.